test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -14,148 +14,126 @@ tags:
+    for i in {1..X}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>; sleep Y; done
     ```
 
-    Replace `numRequests` and `interval` for the values from the following table:
-
-    > These figures are obtained to make the alert fire for ~2 minutes, by calculating
-    > the interval between requests to keep a rate of n request/minute where n
-    > is in the range for the alert
-    >
-    > _Example_: To trigger level 1 we use a rate of 85 (between 80% and 90% of the limit),
-    > so we perform 170 requests every 0.71s. That is:
-    >
-    > - `numRequests = expected rate x minutes to keep the alert firing`
-    > - `interval = 60 / expected rate`
+    While the loop is running: open the Prometheus console:
 
-    | Alert to fire                        | `numRequests` | `interval` |
-    | ------------------------------------ | ------------- | ---------- |
-    | RHOAMApiUsageLevel1ThresholdExceeded | 170           | 0.71       |
-    | RHOAMApiUsageLevel2ThresholdExceeded | 186           | 0.645      |
-    | RHOAMApiUsageLevel3ThresholdExceeded | 240           | 0.5        |
+    ```sh
+    open "https://$(oc get routes prometheus-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+    ```
 
-    > Only the requests for level 3 should be eventually failing with a `429 Too Many Requests` status code
-    > due to the rate limit being exceeded
+    Go to `Alerts` and search for the corresponding alert based on the table above
 
-13) Run the following command and log in to Prometheus service (as a kubeadmin)
+    Click on the alert name -> click on the expression (link) -> Graph
 
-```sh
-open "https://$(oc get routes prometheus-route -n redhat-rhmi-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
-```
+    Verify that after a minute the alert graph is showing some data
 
-14. Go to Alerts -> and search for corresponding the alert (Alert to fire in the table)
-15. Click on the alert name -> click on the expression (link) -> Graph
-16. Run the corresponding script to trigger the alert
-17. Verify that after a minute the alert graph is showing some data
     > In order to verify the current rate, run the following query in the Graph tab
     >
     > ```
-    > increase(ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits[1m]))
+    > increase(ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits[1m])
     > ```
     >
     > The alert should fire when the value is in the expected range
-18. Repeat the same process for the remaining alerts and verify that they are firing as expected. Make sure to wait at least a minute before running each script.
+
+13. Once each of the above `for loops` have completed, verify that a `FIRING` and associated `RESOLVED` email notification is sent. Check the `to` field of the email to ensure that it matches the `recipients` listed in the table above.
```

Comment:
@pmccarthy I was chatting to @laurafitzgerald about the customer grafana dashboard - that we don't have it covered properly with a test case and I suggested to include the verification steps to this test case, since it perfectly connects to it.

the verification steps would be:
1. Open customer grafana dashboard
```
open "https://$(oc get routes grafana-route -n redhat-rhoam-customer-monitoring-operator -o jsonpath='{.spec.host}')"
```
2. Select the **Rate Limiting** dashboard
3. Run the script above with the following configuration:`export numRequests=10000 interval=0.1`
4. Check the graphs are updating and showing the correct data (you should see more than 90% of requests being rejected)

![Screenshot 2020-11-13 at 12 46 44](https://user-images.githubusercontent.com/4881144/99069295-5e34fc00-25ae-11eb-874b-3cf5c4a03736.png)

wdyt?

Reply:
Section has been added 👍